### ISSUE TITLE
Make it an HTTPS server - getUserMedia() not working on insecure origins

### ISSUE THREAD

**Issue by PierreFritsch** [googlecodelabs/webrtc-web#32]
Hi folks

getUserMedia() does not work on insecure origins. See https://goo.gl/rStTGz for more details.

Developers going through step [7. Set up a signaling service to exchange messages](https://codelabs.developers.google.com/codelabs/webrtc-web/#6) of the codelab might want to test how the application behaves across computers.

I'd advocate the codelab should consider switching the application to a secure origin, such as HTTPS.

**Comment by i3games** [comment]
I advocate the same because it is now broken completely without SSL.

**Comment by nitobuendia** [comment]
@PierreFritsch This works for localhost/ on http as per the same doc: "localhost is treated as a secure origin over HTTP, so if you're able to run your server from localhost, you should be able to test the feature on that server."

I understand you mean for when running on a server. Is that correct?
I have seen other issues with similar and recent requests; like #48.

**Comment by PierreFritsch** [comment]
@nitobuendia yes, I understand that the signalling can be tested on localhost.

My point was that as soon as you want to test it with clients on 2 distinct computers, e.g. to check if your network setup allows you to transport the sound, you'll face the issue that at least one browser will block the application because of the insecure origin.

**Comment by nitobuendia** [comment]
A quick update here: we are working on fixing this by serving the codelab on https so it can be tested across devices. There are a few things that need to be fixed first, hence it might take a bit, but on the pipeline. Thank you for reporting and the patience here.

**Comment by chrwei** [comment]
at the very least the guide should have a note about requiring HTTPS and a link to node's docs for setting it up it.
